modules/test/src/main/java/com/devonfw/module/test/common/base/BaseTest.java

```diff
@@ -43,7 +43,7 @@
    * Indicates if the test class is to be set up for the first time. {@code true} indicates that the class has already
    * been set up (e.g., database setup) for the execution of an preceding test method.
    */
-  protected static boolean INITIALIZED = false;
+  protected boolean initialized = false;
```

Comment:
You can not solve this non-static:
JUnit creates a new instance of the class for each test run (method).
Otherwise this problem would have been very easy to solve.

Reply:
JUnit creating a new instance for every test was actually new to me and you're right, this doesn't fix the problem.